### ISSUE TITLE
Consider: keyboard navigation on lists.

### ISSUE THREAD

**Issue by MelSumner** [MelSumner/a11y-automation#398]
For now, I've removed the potential violation related to keyboard navigation for lists. 

"Lists should be navigable with the UP and DOWN ARROW keys" 

But is this accurate? 

Lists are not eligible for interaction, and there are some linting rules that prevent this very thing. Conversely, there's the MDN article about adding keyboard navigation to lists. https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets

Determine if this removal (PR #397 ) was correct, or if this is not only permissible but valid and should be re-added.

**Comment by scottaohara** [comment]
what did "list" mean here?  if a static list, e.g., one rendered by menu, ul, or ol elements in html, or ARIA role=list, then the removal of this seems a good thing, as it would have otherwise been promoting false assertions.

if "list" means a 'listbox', or is a more universal term that can be associated with treeviews or even data tables, then there is some validity to it.... but as is often the case, people use general umbrella terms rather than specify exactly what they mean, and thus uncertainty like this occurs.
